### PR TITLE
make FiltersBar more generic

### LINKED ISSUE #1853: Refactor FiltersBar component to be generic, create VideoFiltersBar and NFTFiltersBar
**Context**
Filters on the Channel > NFTs tab

**AC**
- [ ] Generic component for filtering with shared functionalities
- [ ] Separate component for Video filtering
- [ ] Separate component for NFT filtering
  - [ ] Filter by Status, multiple select [English Auction; Open Auction; Fixed Price; Not for Sale]
  - [ ] Filter by Categories - inherited from Videos

**Designs**
- [ ]  Design of filters adheres to same pattern as for NFTs on members profile. [Designs here > ](https://www.figma.com/file/yhZpTHdf1sxJx13uRZ71GV/Membership-profile?node-id=972%3A229696)

**⛔  OOS**
- [ ] Filter by Price (+ validation of fields)

### PR DESCRIPTION
**Pull request by rafalpawlow**
Closes #1853 
I've tried a little bit specific approach but I find it clean and simple. Anyway, let me know what you think.

### CODE REVIEW

File: src/components/FiltersBar/FiltersBar.tsx [hunk 1]
```diff
@@ -74,112 +95,153 @@ export const FiltersBar: React.FC<ReturnType<typeof useFiltersBar> & FiltersBarP
...
+    () => (
+      <FilterContentContainer>
+        {nftStatuses.map((status) => (
           <Checkbox
-            name="category-filter"
-            label={category.name as string}
-            key={`category-filter-${category.id}`}
-            value={!!categoriesFilter?.includes(category.id)}
+            name="nft-status"
+            label={status.name as string}
```
Comment (eldiegod) [inline src/components/FiltersBar/FiltersBar.tsx]: is the type casting needed here?

File: src/views/viewer/SearchView/SearchResults/SearchResults.tsx [hunk 2]
```diff
@@ -99,7 +97,7 @@ export const SearchResults: React.FC<SearchResultsProps> = React.memo(({ query }
             )}
           </FiltersWrapper>
         </PaddingWrapper>
-        <FiltersBar {...filtersBarLogic} categories={categories} />
+        <FiltersBar {...filtersBarLogic} activeFilters={['date', 'length', 'other', 'language']} />
```
Comment (eldiegod) [inline src/views/viewer/SearchView/SearchResults/SearchResults.tsx]: This one no longer supports categories filter?

**Review by eldiegod — CHANGES_REQUESTED** [review]
Looks clean, added a couple doubts I had!

File: src/views/viewer/SearchView/SearchResults/SearchResults.tsx (hunk 2 above)
Comment (rafalpawlow) [reply src/views/viewer/SearchView/SearchResults/SearchResults.tsx]: No, my bad!

**Review by eldiegod — APPROVED** [review]

File: packages/atlas/src/components/FiltersBar/FiltersBar.stories.tsx [hunk 3]
```diff
@@ -21,16 +23,24 @@ import { useFiltersBar } from './useFiltersBar'
...
+            <Story />
+          </OverlayManagerProvider>
+        </ApolloProvider>
+      )
+    },
   ],
 } as Meta
 
-const RegularTemplate: Story<FiltersBarProps> = () => {
+const RegularTemplate: Story<FiltersBarProps> = (args) => {
```
Comment (kdembler) [inline packages/atlas/src/components/FiltersBar/FiltersBar.stories.tsx:34]: Could you clean this story up? Currently it has unnecessary stuff there other than just the filters bar and also it seems the filters are unusable. When I click on specific filters I don't get any popup

**Review by kdembler — CHANGES_REQUESTED** [review]
One small comment on Storybook, otherwise looks good

**Review by kdembler — APPROVED** [review]

**Pull request merged**